src/styles/main.scss

```diff
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
-}
+@import "../utils/reset";
+@import "../utils/variables";
+@import "../blocks/page";
+@import "../blocks/head";
+@import "../blocks/nav";
+@import "../blocks/catalog";
+@import "../blocks/card";
+@import "../blocks/stars";
```

Comment:
you should move these directories to your `styles` directory

Reply:
Good suggestion !  Code will be come more userfriendly . Done